docs/api.md

```diff
@@ -87,21 +89,22 @@ render, so please only request it if you are actually using it. Be prepared for
-If the `Accept` header is valid, the `Content-Type` header in the response will be set appropriately and the run will be rendered in the specified format, if
+If the `Accept` header is valid, the `Content-Type` header in the response will be set appropriately and the run will be rendered in the specified format. If
 an invalid `Accept` header is supplied, the response `Content-Type` header will be `application/json`, and the status code will be a 406.
 In the 406 reponse there will be an array of values that can be rendered.
 
-| `Accept` Headers Supported       | Return Format      | Return `Content-Type`                 |
-|---------------------------------:|:-------------------|:---------------------------------     |
-| None                             | JSON               | `application/json`                    |
-| `application/json`               | JSON               | `application/json`                    |
-| `application/wsplit`             | WSplit             | `application/wsplit`                  |
-| `application/time-split-tracker` | Time Split Tracker | `application/time-split-tracker`      |
-| `application/splitterz`          | SplitterZ          | `application/splitterz`               |
-| `application/livesplit`          | LiveSplit          | `application/livesplit`               |
-| `application/urn`                | Urn                | `application/urn`                     |
-| `application/llanfair-gered`     | Llanfair-Gered     | `application/llanfair-gered`          |
-| `application/original-timer`     | Original Run File  | One of the following `Content-Type`'s |
+| `Accept` Headers Supported       | Return Format              | Return `Content-Type`                 |
+|---------------------------------:|:---------------------------|:--------------------------------------|
+| None                             | JSON                       | `application/json`                    |
+| `application/json`               | JSON                       | `application/json`                    |
+| `application/splitsio`           | Splits I/O Exchange Format | `application/splitsio`                |
+| `application/wsplit`             | WSplit                     | `application/wsplit`                  |
+| `application/time-split-tracker` | Time Split Tracker         | `application/time-split-tracker`      |
+| `application/splitterz`          | SplitterZ                  | `application/splitterz`               |
+| `application/livesplit`          | LiveSplit                  | `application/livesplit`               |
+| `application/urn`                | Urn                        | `application/urn`                     |
+| `application/llanfair-gered`     | Llanfair-Gered             | `application/llanfair-gered`          |
+| `application/original-timer`     | Original Run File          | One of the following `Content-Type`'s |
```

Comment:
worstrun and source-live-timer are missing here, not sure if intended.

Reply:
Yeah this is intended, this table is a list of exportable file formats, and worstrun and source-live-timer are not exportable.